core/kazoo_speech/src/tts/kazoo_tts_google.erl

```diff
@@ -87,18 +87,16 @@ make_request(Text, Voice, GoogleFormat, Options, EngineData) ->
             create_response(Response, EngineData)
     end.
 
--spec decode_responce(kz_term:ne_binary(), kz_json:object()) -> {kz_term:ne_binary(), kz_term:ne_binary()}.
-decode_responce(Content, EngineData) ->
-    {BinaryContent, NewRequestMeta} = decode_responce(Content, EngineData, []),
-    ContentType = kz_json:get_binary_value(<<"content_type">>, NewRequestMeta),
-    {BinaryContent, ContentType}.
-
--spec decode_responce(kz_term:ne_binary(), kz_json:object(), kz_json:object()) -> {kz_term:ne_binary(), kz_json:object()}.
-decode_responce(Content, EngineData, RequestMeta) ->
-    Base64Result = kz_json:get_binary_value(<<"audioContent">>, kz_json:decode(Content)),
+-spec decode(kz_term:ne_binary(), kz_json:object(), any()) -> {kz_term:ne_binary(), kz_json:object()}.
+decode(Contents, RequestMeta, EngineData) ->
```

Comment:
If the callback's type is `any()`, can we at least spec the implementation's `EngineData` type? :)

Reply:
done